ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/editor/EditorAgentImpl.java

```diff
@@ -275,6 +288,16 @@ protected void onFailure(Throwable exception) {
         });
     }
 
+    private boolean descriptorHasFile(ProjectDescriptor descriptor, VirtualFile virtualFile) {
```

Comment:
ProjectDescriptor doesn't have a files. Please use another name for this method.